src/mono/wasm/Wasm.Build.Tests/Blazor/DebugLevelTests.cs

```diff
@@ -0,0 +1,38 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Xunit;
+using Xunit.Abstractions;
+using Wasm.Build.Tests.TestAppScenarios;
+
+#nullable enable
+
+namespace Wasm.Build.Tests.Blazor;
+
+public class DebugLevelTests : AppTestBase
+{
+    public DebugLevelTests(ITestOutputHelper output, SharedBuildPerTestClassFixture buildContext)
```

Comment:
I think we should merge this one with `Wasm.Build.Tests.DebugLevelTests` and use "WasmBasicTestApp" for all of the tests.

+ Move it outside of Blazor namespace since it's not blazor specific